src/lib.rs

```diff
@@ -90,10 +95,40 @@ impl<Types: IpfsTypes> fmt::Debug for IpfsOptions<Types> {
             .field("ipfs_log", &self.ipfs_log)
             .field("ipfs_path", &self.ipfs_path)
             .field("bootstrap", &self.bootstrap)
+            .field("keypair", &DebuggableKeypair(&self.keypair))
             .finish()
     }
 }
 
+use std::borrow::Borrow;
+
+#[derive(Clone)]
+struct DebuggableKeypair<I: Borrow<Keypair>>(I);
```

Comment:
Note to self: the libp2p type needs to have a Debug impl, not sure why they don't already have one. This is the nicest workaround I could figure out.

Reply:
Can we submit a PR to them?